code/datums/chatmessage.dm

```diff
@@ -56,15 +56,15 @@
   * * italics - Should we use italics or not
   * * lifespan - The lifespan of the message in deciseconds
   */
-/datum/chatmessage/New(text, atom/target, mob/owner, italics, size, lifespan = CHAT_MESSAGE_LIFESPAN)
+/datum/chatmessage/New(text, atom/target, mob/owner, italics, size, lifespan = CHAT_MESSAGE_LIFESPAN, symbol)
```

Comment:
Could you add the new `symbol` variables to the documentation too? Since the other arguments are there already.